pkg/apiutil/apiutil.go

```diff
@@ -117,13 +117,32 @@ func ReadJSONRespondError(rd *render.Render, w http.ResponseWriter, body io.Read
 func ErrorResp(rd *render.Render, w http.ResponseWriter, err error) {
 	if err == nil {
 		log.Error("nil is given to errorResp")
-		rd.JSON(w, http.StatusInternalServerError, "nil error")
+		rd.JSON(w, http.StatusInternalServerError, WrapOutput(http.StatusInternalServerError, "nil error"))
```

Comment:
Maybe more easy to use (don't have to wrapOutput for every response) if we can somehow wrap the `render.Render`?